docs/source/dev-guide/release/release-steps.rst

```diff
@@ -66,7 +66,7 @@ merged since the last release.
        pip install twine  # if not installed
        twine upload dist/* -r pypi
 
-11. A PR is automatically submitted (this will take a few hours) on [`conda-forge/eva-feedstock`](https://github.com/conda-forge/eva-feedstock) to update the version.
```

Comment:
@gaurav274 what is the right link for this one?

Reply:
We can remove it. We don't need it.